index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concert-website</title>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+
+</head>
+
+<body>
+    <section class="top-nav">
+        <div></div>
+        <div class="top-nav-content">
+            <img src="./items/facebook-icon.png" alt="Facebook icon">
+            <img src="./items/twitter-icon.png" alt="twiter icon">
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Logout</a>
+        </div>
+    </section>
+    <header>
+        <a href="index.html" class="logo">
+            <img src="./items/Logo.png" alt="this is the image of the logo">
+        </a>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        <div class="nav-lists">
+            <a href="about.html" class="nav-link">About</a>
+            <a href="" class="nav-link">Schedule</a>
+            <a href="" class="nav-link">Contact-Us</a>
+            <button>Get Ticket</button>
+
+        </div>
```

Comment:
- [ ] Kindly align the bottom `nav` so as to match the given design. You could add some `margin-top` to the `nav-lists` class

Your design
![image](https://user-images.githubusercontent.com/81932478/196937893-a895c610-1810-4ced-a6e5-c642a15fc91d.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196938133-ea987a3f-d977-4aa3-b06c-f22a1afd2c22.png)